man/fi_domain.3.md

```diff
@@ -482,12 +482,19 @@ transfer operation.
   operations, or attempt to access outside of the target memory region
   will fail, resulting in a transmit error.
 
-When a resource management error occurs on an endpoint, the endpoint is
-transitioned into a disabled state.  Any operations which have not
-already completed will fail and be discarded.  For connectionless endpoints,
-the endpoint must be re-enabled before it will accept new data transfer
-operations.  For connected endpoints, the connection is torn down and
-must be re-established.
+When a resource management error occurs on an connected endpoint, the endpoint
+will transition into a disabled state and the connection torn down. While
+transitioning to disabled, any queued and inflight operations will be dropped.
+Connection must be re-established for endpoint to be usable.
```

Comment:
I'd consider removing this sentence.  It may not be possible to re-reconnect the same endpoint, forcing the use of a new one.